overlord/devicestate/firstboot.go

```diff
@@ -42,6 +43,8 @@ var errNothingToDo = errors.New("nothing to do")
 
 var runtimeNumCPU = runtime.NumCPU
 
+var createKnownUsers = createAllKnownSystemUsers
```

Comment:
I don't think this is what @mardy had in mind. I think he was proposing to turn https://github.com/snapcore/snapd/blob/master/overlord/devicestate/users.go#L190 into:

```
var createAllKnownSystemUsers = func(state *state.State, assertDb asserts.RODatabase, model *asserts.Model, serial *asserts.Serial, sudoer bool) ([]*CreatedUser, error) {...
```
itself directly

Reply:
Yes indeed, my point was that creating a variable with slightly different name is confusing. Sorry for not having been clear, but Samuele's suggestion is exactly what I had in mind. :-)